httpx/utils.py

```diff
@@ -107,6 +107,14 @@ def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
     return netrc_info.authenticators(host)  # type: ignore
 
 
+def get_ssl_cert_file() -> typing.Optional[str]:
+    for env_name in ("SSL_CERT_FILE", "SSL_CERT_DIR"):
+        ssl_file = Path(os.getenv(env_name, ""))
+        if ssl_file and ssl_file.is_file():
```

Comment:
This won't work for `SSL_CERT_DIR`, for hopefully obvious reasons.

Reply:
ahh sorry, it's path. I'm fixing